### PR TITLE
Add AWS ALB access log parser

### PR DESCRIPTION
**Pull request by Zaulao**
Adds a parser for Application Load Balancers (ALB) access logs (which is often used as ingress in Kubernetes clusters).

From the raw log, relevant fields for HTTP analysis are extracted. I used the parser for Nginx and the data analyzed by some of the base-http-scenarios as a basis.

### CODE REVIEW

**Comment by Zaulao** [comment]
No problem! Just added it

**Comment by Zaulao** [comment]
Sorry for the typo, I got it wrong after forking the project to prepare de PR

File: .tests/aws-alb-logs/config.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+parsers:
```
Comment (LaurenceJJones) [inline .tests/aws-alb-logs/config.yaml:1]: can we add s00 stage parser

`- crowdsecurity/syslog-logs`

Even though it is not really used it "real runtim" there is always a stage s00 parser

File: .tests/aws-alb-logs/parser.assert [hunk 2]
```diff
@@ -0,0 +1,286 @@
+len(results) == 3
```
Comment (LaurenceJJones) [inline .tests/aws-alb-logs/parser.assert:1]: Once you add stage s00 parser you will need to regen .assert

**Review by LaurenceJJones — CHANGES_REQUESTED** [review]
Small changes but we also need tests for some scenarios. If you can bring up to upstream changes and also do my requested changes I can create the scenarios tests.

**Comment by Zaulao** [comment]
Is there anything else I can do here?

**Review by blotus — APPROVED** [review]

**Comment by blotus** [comment]
Hello,

Sorry about the delay; the PR fell through the cracks.

I'm merging it; thanks for the contribution!

**Pull request merged**